lesson4/lesson4_1/lesson4_1/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace lesson4_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string first = Convert.ToString(Console.ReadLine());
+            string symbol = Convert.ToString(Console.ReadLine());
+            string second = Convert.ToString(Console.ReadLine());
+            
+            try
+            {
+                double x = Convert.ToDouble(first);
+                double y = Convert.ToDouble(second);
+                char symb = Convert.ToChar(symbol);
+                Console.WriteLine("How do you want to make a calculation? :\n1.Through if/else\n2.Through switch/case");
+                string choice = Convert.ToString(Console.ReadLine());
+                byte selected = Convert.ToByte(choice);
+                if (selected == 1) { Calculated(x, y, symb); }
+                if (selected == 2) { Calc(x, y, symb); }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("incorrect input data");
+            }
+            Console.ReadKey();
+        }
+
+        private static void Calculated(double x, double y, char symbol)
+        {
+            string result = "";
+            if (symbol == '+') { result = x + " + " + y + " = " + (x + y); }
+            else if (symbol == '-') { result = x + " - " + y + " = " + (x - y); }
```

Comment:
It's better to use here the latest string formatting feature with $ symbol.
ie. result = $"{x} + {y} = { x + y}".
It's more readable, but it's out of scope for our lessons, so you should work on this yourself.